tests/one-off/sham-install.rkt

```diff
@@ -0,0 +1,59 @@
+#lang racket
+
+(module+ test
+  (require
+   racket/runtime-path
+   rackunit
+   ffi/unsafe
+   sham/ast
+   sham/ir
+   sham/env
+   sham/llvm/ffi/all
+   sham/llvm/llvm-config)
+
+  (llvm-initialize-all)
+  (define context (LLVMContextCreate))
+
+  (define module
+    (env-get-llvm-module
+     (build-llvm-module
+      (dmodule
+       (hash)
+       'module
+       (list
+        (function^
+         'main () i64
+         (return (ui64 42))))))))
+
+  (define-runtime-path basename "./test42")
+  (define bc-path (path-add-extension basename ".bc"))
+  (define bin-path (llvm-config "--bindir"))
+  (define asm-path (path-add-extension basename ".s"))
+
+  (check-equal?
+   0
+   (LLVMWriteBitcodeToFile module (path->string bc-path)))
+
+  (check-true
+   (system*
+    (build-path bin-path "llc")
+    "-O3"
+    "--relocation-model=pic"
+    bc-path))
+
+  (check-true
+   (system*
+    (build-path bin-path "clang")
+    asm-path
+    "-o"
+    basename))
+
+  (check-equal?
+   42
+   (system*/exit-code basename))
```

Comment:
These tests looks like correctness unit tests for sham, is that the intention? Why not run the test suite for sham instead?

Reply:
The intention of the test is to ensure that sham is installed and that the llvm portion is working. Does our install script successfully install sham and if the llvm is present can we generate code. Sham is build around a JIT system which I don't care about working correctly, and in fact doesn't work on my laptop. The sham test suite would check the functionality of the JIT.

Reply:
I see. I think it would be a good idea to write a comment to that extent in the header of this module.